examples/chip-tool/commands/clusters/ModelCommand.h

```diff
@@ -67,11 +70,14 @@ class ModelCommand : public CHIPCommand
 
 protected:
     chip::Optional<uint16_t> mTimeout;
+    chip::Optional<int> mIsPeerLITCLI;
```

Comment:
This member name does not make sense.  It's set from non-CLI all the time...  Why is this not just called `mIsPeerLIT`?

Reply:
updated, thanks

Reply:
Well, since the we have another mIsPeerLIT in `InteractionModelReports`, which is not on the chain of Command classes, but is a parent class of ReportCommands.

I modified this to a private member, and expose via a method to make the code a bit cleaner.